### ISSUE TITLE
The automatic action "telemetry" starts but never ends

### ISSUE THREAD

**Issue by tabad** [glpi-project/glpi#4156]
## Environment

* GLPI v9.2.3
    * FusionInventory for GLPI v9.2+2.0
    * ...and other plugins

------------
* Steps to reproduce (which actions have you made)
    Execute the automatic action "telemetry" (*Home > Setup > Automatic actions > Telemetry > Execute*).

* Expected result 
    The automatic action "telemetry"  will end at some point.

* Actual result
    The automatic action "telemetry" starts but never ends.

* URL of the page
    Not pertinent.

* Screenshot of the problem (if pertinent)
    Not pertinent.

* Your GLPI setup (you can find it in Setup > General menu, System tab)
    It's [here](https://github.com/glpi-project/glpi/files/2087764/glpi-system-information.txt).

------------

## The issue

The automatic action "telemetry" starts but never ends. The system sends an electronic mail informating that an automatic action requires intervention

```text
The following automatic actions are in error. They require intervention.


telemetry : Send telemetry informations
```

 and the error log file of PHP-FPM pool  contains the following messages.

```text
[04-Jun-2018 13:44:23 Europe/Madrid] PHP Fatal error:  Uncaught RuntimeException: Something went wrong sending telemetry informations: No data available on the web site in /usr/local/share/glpi/inc/telemetry.class.php:290
Stack trace:
#0 /usr/local/share/glpi/inc/crontask.class.php(832): Telemetry::cronTelemetry(Object(CronTask))
#1 /usr/local/share/glpi/front/cron.php(65): CronTask::launch(1)
#2 {main}
  thrown in /usr/local/share/glpi/inc/telemetry.class.php on line 290
[04-Jun-2018 18:31:49 Europe/Madrid] PHP Fatal error:  Uncaught RuntimeException: Something went wrong sending telemetry informations: No data available on the web site in /usr/local/share/glpi/inc/telemetry.class.php:290
Stack trace:
#0 /usr/local/share/glpi/inc/crontask.class.php(832): Telemetry::cronTelemetry(Object(CronTask))
#1 /usr/local/share/glpi/front/crontask.form.php(49): CronTask::launch(-1, 1, 'telemetry')
#2 {main}
  thrown in /usr/local/share/glpi/inc/telemetry.class.php on line 290
[08-Jun-2018 23:18:07 Europe/Madrid] PHP Fatal error:  Uncaught RuntimeException: Something went wrong sending telemetry informations: No data available on the web site in /usr/local/share/glpi/inc/telemetry.class.php:290
Stack trace:
#0 /usr/local/share/glpi/inc/crontask.class.php(832): Telemetry::cronTelemetry(Object(CronTask))
#1 /usr/local/share/glpi/front/crontask.form.php(49): CronTask::launch(-1, 1, 'telemetry')
#2 {main}
  thrown in /usr/local/share/glpi/inc/telemetry.class.php on line 290
[10-Jun-2018 11:50:22 Europe/Madrid] PHP Fatal error:  Uncaught RuntimeException: Something went wrong sending telemetry informations: No data available on the web site in /usr/local/share/glpi/inc/telemetry.class.php:290
Stack trace:
#0 /usr/local/share/glpi/inc/crontask.class.php(832): Telemetry::cronTelemetry(Object(CronTask))
#1 /usr/local/share/glpi/front/crontask.form.php(49): CronTask::launch(-1, 1, 'telemetry')
#2 {main}
  thrown in /usr/local/share/glpi/inc/telemetry.class.php on line 290
```

The communication of the system with Internet is correct; *Home > Administration > Maintenance > Check if a new version is available* informs with "You have the latest available version".

Regards,

**Comment by tabad** [comment]
Hello again,

GLPI is running behind a proxy server (I don't have access to it, sorry) but, as I said,  *Home > Administration > Maintenance > Check if a new version is available* works perfectly.

Taking a look to #3559 I have see that `toolbox.class.php` may be related. Searching for some kind of error regarding Toolbox in `php-errors.log` file I have found the following.

```text
2018-06-08 23:18:07 [7@inventario.sescam.jclm.es]
  From Toolbox::callCurl() in /usr/local/share/glpi/inc/toolbox.class.php line 1652
  No data available on the web site
...
...
2018-06-10 11:50:22 [7@inventario.sescam.jclm.es]
  From Toolbox::callCurl() in /usr/local/share/glpi/inc/toolbox.class.php line 1652
  No data available on the web site
```

I hope this information help you. Regards,

**Comment by orthagh** [comment]
Hello @tabad 

Thank you for your report.
We are focus on 9.3 release atm.

After that, we (with @trasher ) will migrate the telemetry to a new server and to the last version of it.
If you are alright, i'll ping you at this moment.

**Comment by orthagh** [comment]
should be fixed by glpi-project/telemetry/pull/53

**Comment by tabad** [comment]
Yes, it works perfectly now.

Thanks!.
